apis/postgresql/v1alpha1/provider_types.go

```diff
@@ -58,7 +58,7 @@ type ProviderConfigStatus struct {
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:name="AGE",type="date",JSONPath=".metadata.creationTimestamp"
 // +kubebuilder:printcolumn:name="SECRET-NAME",type="string",JSONPath=".spec.credentialsSecretRef.name",priority=1
-// +kubebuilder:resource:scope=Cluster
+// +kubebuilder:resource:scope=Cluster,categories={crossplane,managed,sql}
```

Comment:
nitpick, we typically don't consider the `ProviderConfig` resources a managed resource. We have used category `provider` elsewhere (ref: https://github.com/crossplane/provider-aws/blob/211fada49e7d42cbed6035ef04e1e55ace941e27/apis/v1beta1/providerconfig_types.go#L50)
```suggestion
// +kubebuilder:resource:scope=Cluster,categories={crossplane,provider,sql}
```